examples/src/main/scala/examples/hybrid/state/HBoxStoredState.scala

```diff
@@ -121,6 +121,7 @@ object HBoxStoredState {
           val forgerNonce = Nonce @@ Longs.fromByteArray(ps.id.take(8))
           val forgerBox = PublicKey25519NoncedBox(ps.generatorBox.proposition, forgerNonce, Value @@ reward)
 
+          @SuppressWarnings(Array("org.wartremover.warts.Product","org.wartremover.warts.Serializable"))
```

Comment:
Because the Scala compiler probably infers "Product with Serializable" instead of `BoxStateChangeOperation` and adding types annotations to solve this probably looks too verbose.